cmd/xl-v1-utils_test.go

```diff
@@ -17,12 +17,15 @@
 package cmd
 
 import (
+	"encoding/hex"
 	"encoding/json"
 	"errors"
 	"reflect"
 	"strconv"
 	"testing"
 
+	"bytes"
```

Comment:
Looks like these are wrongly placed imports.. stdlib needs to be together perhaps move this up.